helm/postgres/templates/postgres.yaml

```
@@ -21,6 +21,7 @@ spec:
     - name: {{ default "instance1" .Values.instanceName | quote }}
       replicas: {{ default 1 .Values.instanceReplicas }}
       dataVolumeClaimSpec:
+        {{- if .Values.storageClassName -}}storageClassName: {{ .Values.storageClassName | quote }}{{ end }}
```

```suggestion
        {{- if .Values.instanceStorageClassName }}
        storageClassName: {{ .Values.instanceStorageClassName | quote }}
        {{- end }}
```
A) changing the spacing here and below to keep with the pattern already established.
B) `values.yaml` has a `instanceStorageClassName` field, not. `storageClassName` field